src/CLI/actioner/sonic-cli-udld.py

```diff
@@ -30,41 +30,104 @@ def invoke(func, args):
     body = None
     aa = cc.ApiClient()
 
-    if func == 'udldGlobalShowHandler':
-        return udldShowHandler("global_show", args)
-
-    if func == 'udldNeighborShowHandler':
-        return udldShowHandler("neighbor_show", args)
-
-    if func == 'udldInterfaceShowHandler':
-        return udldShowHandler("interface_show", args)
-
-    if func == 'udldInterfaceCountersShowHandler':
-        return udldShowHandler("counters_show", args)
-
-    # Configure UDLD global
-    if func == 'udldGlobalEnableHandler' :
-        return udldConfigHandler("global_enable", args)
-
-    # Configure UDLD normal
-    if func == 'udldGlobalNormalEnableHandler' :
-        return udldConfigHandler("global_normal", args)
+    # show udld global
+    if func == 'get_sonic_udld_sonic_udld_udld_udld_list':
+        keypath = cc.Path('/restconf/data/sonic-udld:sonic-udld/UDLD/UDLD_LIST={id}', id='GLOBAL')
```

Comment:
The uuld is using the Request package to send the REST request. But it is noted that others like mirroring etc. directly attempts to the Redis DB. Why not consistently implementing CLI actioner with REST api?